doc/generate.py

```diff
@@ -54,12 +66,13 @@ def generate(klass):
     * `id`: str, required
     * `dataset`: Dataset, required
     """
-    doc = [f'# {klass.type} connector', doc_or_empty(klass), '## Data provider configuration']
```

Comment:
Updates in this function are not related to the importation issue also right ? If so, maybe you can commit it in an other commit and describe why you are doing these change in the commit message ?

Reply:
I did a separate commit and provided clarifications about the change.